src/screens/ContactScreen/index.js

```diff
@@ -317,10 +382,9 @@ const ContactScreen = ({navigation}) => {
     if (changedText.length > 0) {
       debounced(changedText);
     } else {
+      setUsersSearch([]);
       debounced.cancel();
     }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onSearchTextChange` function has been updated to clear the search results when the search text is cleared. This is a good change as it improves the user experience by not showing stale search results.



<!-- This is an auto-generated comment by OSS CodeRabbit -->